ui/allrequest.html

```diff
@@ -0,0 +1,75 @@
+!DOCTYPE html>
```

Comment:
Is this file necessary for this pull request? @OKiMaureen

Reply:
no, It was a mistake